compiler/rustc_mir/src/interpret/eval_context.rs

```diff
@@ -134,14 +134,23 @@ pub struct FrameInfo<'tcx> {
     pub lint_root: Option<hir::HirId>,
 }
 
-#[derive(Clone, Eq, PartialEq, Debug, HashStable)] // Miri debug-prints these
+/// Unwind information.
+#[derive(Clone, Copy, Eq, PartialEq, Debug, HashStable)]
+pub enum StackPopUnwind {
+    /// The cleanup block.
+    Cleanup(Option<mir::BasicBlock>),
+    /// Unwinding is not allowed (UB).
+    NotAllowed,
+}
+
+#[derive(Clone, Copy, Eq, PartialEq, Debug, HashStable)] // Miri debug-prints these
```

Comment:
I have to add `Copy` here, otherwise I will get this error (for https://github.com/rust-lang/rust/pull/85546#discussion_r637156732):
```
error[E0382]: use of partially moved value: `frame`
   --> compiler/rustc_mir/src/interpret/eval_context.rs:845:37
    |
818 |         let (cleanup, next_block) = match (frame.return_to_block, unwinding) {
    |                                            --------------------- value partially moved here
...
845 |         if M::after_stack_pop(self, frame, unwinding)? == StackPopJump::NoJump {
    |                                     ^^^^^ value used here after partial move
    |
    = note: partial move occurs because `frame.return_to_block` has type `StackPopCleanup`, which does not implement the `Copy` trait
```
(Or, maybe I have made a mistake when writing the `match`...)

Reply:
Making this type `Copy` seems totally fine for me.